package.json

```diff
@@ -59,7 +59,7 @@
     "@types/cors": "2.8.12",
     "@types/express": "4.17.14",
     "@types/express-serve-static-core": "4.17.31",
-    "@types/jest": "28.1.8",
+    "@types/jest": "29.0.2",
```

Comment:
If you only use `@jest/globals` you can remove this

Reply:
Ah, good catch. Missed this even though I removed it from the tsconfig 😄